agent/bdiagent.py

```diff
@@ -38,32 +38,32 @@ def getpercepts(self, beliefbase):
 
     def manage_goals(self, beliefbase, goalbase):
         for goal in self.goalbase:
-            if (self.is_achieved(goal)):
+            if self.is_achieved(goal):
                 print(goal, " Goal Achieved!")
                 self.achieved_goal(goal, goalbase)
         return
 
     # A goal is achieved if either it shares a name with a belief and the belief is true or its associated function returns true
     def is_achieved(self, goal):
-        if (goal in self.beliefbase.keys()):
-            if (self.beliefbase[goal] == 1):
+        if goal in self.beliefbase.keys():
```

Comment:
You don't actually need the .keys call here the in operator does this check automatically with dicts

Reply:
matches my testing, I'll update it cheers